communities.md

```diff
@@ -122,6 +122,7 @@ The supported parameters are:
 * **(mandatory)** parent, the UUID of the parent community
 * page, size [see pagination](README.md#Pagination)
 
+<<<<<<< HEAD
```

Comment:
There's a random merge conflict HEAD note here ;)

Reply:
Thanks. Fixed in latest rebase.